<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCTableDialog</class>
 <widget class="QDialog" name="CCTableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCTableDialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/cctable.png</normaloff>Resources/cctable.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CCTableView" name="tWCCTableView">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>438</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pBInit">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Initialize cc dialog</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MyEditorEBD.qrc">
           <normaloff>:/MyEditorEBD/Resources/btn_refresh.png</normaloff>:/MyEditorEBD/Resources/btn_refresh.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pBSave">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidgets">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab { 
  background: lightgray;
  color: black;
    padding: 20px;
}
QTabBar::tab:selected {
  background: white;
 }</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabNormal">
        <attribute name="title">
         <string>Normal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="NormalWidget" name="wDNormal" native="true">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabChenile">
        <attribute name="title">
         <string>Chenile</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="ChenileWidget" name="wDChenile" native="true">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabTape">
        <attribute name="title">
         <string>Tape</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="TapeWidget" name="wDTape" native="true">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>58</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Add and Remove</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pBAdd">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MyEditorEBD.qrc">
              <normaloff>:/MyEditorEBD/Resources/btn_needle_add.png</normaloff>:/MyEditorEBD/Resources/btn_needle_add.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pBRemove">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MyEditorEBD.qrc">
              <normaloff>:/MyEditorEBD/Resources/btn_needle_del.png</normaloff>:/MyEditorEBD/Resources/btn_needle_del.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NormalWidget</class>
   <extends>QWidget</extends>
   <header>NormalWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChenileWidget</class>
   <extends>QWidget</extends>
   <header>ChenileWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TapeWidget</class>
   <extends>QWidget</extends>
   <header>TapeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CCTableView</class>
   <extends>QTableView</extends>
   <header>CCTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MyEditorEBD.qrc"/>
 </resources>
 <connections/>
</ui>
